conf/messages

```diff
@@ -127,15 +126,12 @@ AddBenefits.CY.Caution.p12.3 = (starting 6 April {0})
 
 AddBenefits.noselection.error = You need to select at least 1 benefit or expense before you can continue
 
-AddBenefits.Confirm.Single.Title = Register a benefit or expense
-AddBenefits.Confirm.Multiple.Title = Register benefits and expenses
+AddBenefits.Confirm.Single.Title = Check the benefits and expenses you want to register
```

Comment:
I don't see use of the key `AddBenefits.Confirm.Single.Title` anywhere in the code so remove from here